fusion/README.md

```diff
@@ -13,22 +13,22 @@ ansible-playbook playbook_name.yml
 In almost every playbook, there are 2 lines that require an environment variable
 
 ```
-app_id: "{{ ansible_env.API_CLIENT}}"
-key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+issuer_id: "{{ ansible_env.ISSUER_ID}}"
+private_key_file: "{{ ansible_env.PRIVATE_KEY_FILE}}"
 ```
-In this case, the variables are: API_CLIENT and PRIV_KEY_FILE
+In this case, the variables are: ISSUER_ID and PRIVATE_KEY_FILE
 To set them, you can temporarily enable them with:
 ```
-export API_CLIENT='pure1:apikey:123456789'
-export PRIV_KEY_FILE='/home/user/key.pem'
+export ISSUER_ID='pure1:apikey:123456789'
+export PRIVATE_KEY_FILE='/home/user/key.pem'
 ```
-in the case of ```PRIV_KEY_FILE```, the path to the ```key.pem``` need to be aboslute.
+in the case of ```PRIVATE_KEY_FILE```, the path to the ```key.pem``` needs to be absolute.
 
 If you prefer to not use environment variables, you can change the values inside the playbook:
 
 ```
-app_id: "<your_API_Application_ID_here>"
-key_file: "/home/user/key.pem"
+issuer_id: "<your_API_Issuer_ID_here>"
```

Comment:
Maybe we can add description of alternative and additional fields
```
If you are using an encrypted key, you can use `private_key_password` field or `FUSION_PRIVATE_KEY_FILE` to specify the private key password

You can change Fusion API host and token endpoint by specifying FUSION_API_HOST  and FUSION_TOKEN_ENDPOINT env vars
```

```
As alternative you can use `FUSION_ACCESS_TOKEN` env var or `access_token` field to specify API's access token directly.
```

Reply:
Unfortunately, this repo does not mention neither FUSION_ISSUER_ID nor FUSION_PRIVATE_KEY_FILE, instead suggests the use of two non-standard env vars (ISSUER_ID and PRIVATE_KEY_FILE). Adding the env vars you suggest without making the rest consistent would be confusing.

Reply:
I suggest we refactor the samples to use the standard `FUSION_PRIVATE_KEY_FILE` and `FUSION_ISSUER_ID` env vars. Although these are recognized by the module, so technically it is unneccessary to set them as params, this would make it consistent with what we have in [Fusion-Collection](https://github.com/Pure-Storage-Ansible/Fusion-Collection/blob/master/playbooks/simple/create_array.yml#L8) and would also reduce the confusion when introducing the env vars you suggest.

Reply:
We described how to use fields in README. So you are right, I guess we refactor samples.